### ISSUE TITLE
通过summary打印出模型的Total params:和Params size (MB)与timm库中的bit_base模型不一样，理论上相同的设置应该是一样，这是什么原因呢？

### ISSUE THREAD

**Issue by lucker26** [lucidrains/vit-pytorch#329]
import torch
from vit import ViT
from torchsummary import summary
import timm
v = ViT(
    image_size = 224,
    patch_size = 16,
    num_classes = 1000,
    dim = 768,
    depth = 12,
    heads = 12,
    mlp_dim = 3072,
    dropout = 0.1,
    emb_dropout = 0.1
)

# 使用 summary 显示模型的摘要
summary(v, input_size=(3, 224, 224), device='cpu')  # 传入输入的形状 (C, H, W)

# 加载 ViT-B/16 模型
model = timm.create_model('vit_base_patch16_224', pretrained=False)

# 打印模型的摘要信息
summary(model, input_size=( 3, 224, 224))

**Comment by lucker26** [comment]
![image](https://github.com/user-attachments/assets/01374558-3275-4181-89f6-0b610befdc3f)
The result of the first diagram is the model you wrote. The result of the second graph is timm's.
![image](https://github.com/user-attachments/assets/46bf2b52-f7cf-44fb-9f83-aa2d2d401063)
